contributing/code/bc.rst

```diff
@@ -253,6 +257,10 @@ Change return type                                  Yes
 **Static Methods**
 Turn non static into static                         No
 Turn static into non static                         No
+**Constants**
+Add constant                                        Yes
+Remove constant                                     No
+Change value of a constant                          Yes [5]_
```

Comment:
I agree too, since constant has public access, people should know about it's change.